src/app/_components/Form/FormLabel.tsx

```diff
@@ -3,35 +3,49 @@ import React, { type ElementType } from 'react'
 import { Label } from '@headlessui/react'
 import { clsx } from 'clsx'
 
+import {
+  FormLabelDescription,
+  type FormLabelDescriptionProps,
+} from './FormLabelDescription'
+
 export type FormLabelProps = {
+  as?: ElementType
   label: string
   hideLabel?: boolean
   addOptionalToLabel?: boolean
-  as?: ElementType
+  description?: FormLabelDescriptionProps['children']
 }
 
 export function FormLabel({
+  as = 'label',
   label,
   hideLabel = false,
   addOptionalToLabel = false,
-  as = 'label',
+  description,
   ...rest
 }: FormLabelProps) {
   return (
-    <Label
-      {...rest}
-      as={as}
-      className={clsx(
-        'inline-block text-base font-bold capitalize text-brand-100',
-        hideLabel && 'sr-only',
-      )}
-    >
-      {label}
-      {addOptionalToLabel && (
-        <span className="ml-2 inline-block font-light lowercase">
-          (optional)
+    <span>
+      <Label
+        {...rest}
+        as={as}
+        className={clsx(
+          'inline-block text-base font-bold text-brand-100',
+          hideLabel && 'sr-only',
+        )}
+      >
+        {label}
+        {addOptionalToLabel && (
+          <span className="ml-2 inline-block font-light lowercase">
+            (optional)
+          </span>
+        )}
+      </Label>
+      {description && (
+        <span className="my-1 block">
```

Comment:
What the margin-bottom for? Should it be part of this component?

Reply:
Check if we can abstract FormDescription away from FormLabel.

Reply:
✅

Reply:
Much better! `children: React.ReactNode` - Is it a simple `string`?

Reply:
We also use the description for the file input, where it's a ReactNode.

```ts
description={
  <>
    For best quality, please submit a white logo with a transparent
    background, at least 1000px by 1000px, and under{' '}
    {prettyBytes(MAX_FILE_SIZE_IN_BYTES)}. You can use tools like{' '}
    <ExternalTextLink
      href="https://squoosh.app/"
      target="_blank"
      rel="noreferrer noopenner"
    >
      Squoosh
    </ExternalTextLink>{' '}
    to compress your image.
  </>
}
```